pkg/abcupdater/version.go

```diff
@@ -252,3 +273,39 @@ func updateVersionOutput(updateDetails *versionUpdateDetails) (string, error) {
 
 	return b.String(), nil
 }
+
+func (c *CheckVersionParams) loadLocalCachedData() (*LocalVersionData, error) {
+	var path string
+	if c.CacheFileOverride != "" {
+		path = c.CacheFileOverride
+	} else {
```

Comment:
You can avoid the "else" here using the same pattern from line 101:

```go
path := c.CacheFileOverride
if path == "" {
  // contents of else block go here
}
```

Same in setLocalCachedData.

Reply:
nice